cluster-scope/overlays/prod/moc/smaug/kustomization.yaml

```diff
@@ -138,6 +140,8 @@ resources:
 - ../../../../base/storage.k8s.io/storageclasses/moc-nfs-csi
 - ../../../../base/storage.k8s.io/storageclasses/ocs-external-storagecluster-ceph-rbd
 - ../../../../base/user.openshift.io/groups/cluster-admins
+- ../../../../base/user.openshift.io/groups/okd-team
+- ../../../../base/user.openshift.io/groups/okd-wg
```

Comment:
Can you add the groups here instead [1](https://github.com/operate-first/apps/blob/master/cluster-scope/overlays/prod/common/kustomization.yaml)